idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/column.py

```diff
@@ -0,0 +1,1799 @@
+
+        # ---------------------------------------------------------------------
+        # Deactivate unit model level constraints (asides geometry constraints)
+        for c in blk.component_objects(Constraint, descend_into=True):
+            if blk.config.dynamic:
+                if c.local_name in dynamic_constraints:
+                    c.deactivate()
+            elif not blk.config.dynamic:
+                if c.local_name in steady_state_constraints:
+                    c.deactivate()
+
+        # For some reasons E1_eqn & E2_eqn  are still active
+        # So deactivating again.
+        blk.E1_eqn.deactivate()
+        blk.E2_eqn.deactivate()
+        # Fix some variables
+        # Hydrodynamics - velocity
+        blk.vel_liq[:, :].fix()
+        blk.vel_vap[:, :].fix()
+        # interface pressure
+        blk.pressure_equil[:, :, :].fix()
+        # flux
+        blk.N_v[:, :, :].fix(0.0)
+        blk.vapor_phase.mass_transfer_term.fix(0.0)
+        blk.liquid_phase.mass_transfer_term.fix(0.0)
+        # Enhancement factor model
+        blk.E[:, :].fix()
+        blk.yi_MEA[:, :].fix()
+        blk.yeq_CO2[:, :].fix()
+        # heat transfer
+        blk.Q_v[:, :].fix(0.0)
+        blk.Q_l[:, :].fix(0.0)
+        blk.vapor_phase.heat[:, :].fix(0.0)
+        blk.liquid_phase.heat[:, :].fix(0.0)
+        # area
+        blk.vapor_phase.area.fix(1)
+        blk.liquid_phase.area.fix(1)
+
+        # other variables
+        # Pressure_dx
+        blk.vapor_phase.pressure_dx[:, :].fix(0.0)
+        # vapor side flow terms
+        blk.vapor_phase._enthalpy_flow[:, :, :].fix(1.0)
+        blk.vapor_phase.enthalpy_flow_dx[:, :, :].fix(0.0)
+        blk.vapor_phase._flow_terms[:, :, :, :].fix(1.0)
+        blk.vapor_phase.material_flow_dx[:, :, :, :].fix(0.0)
+        # liquid side flow terms
+        blk.liquid_phase._enthalpy_flow[:, :, :].fix(1.0)
+        blk.liquid_phase.enthalpy_flow_dx[:, :, :].fix(0.0)
+        blk.liquid_phase._flow_terms[:, :, :, :].fix(1.0)
+        blk.liquid_phase.material_flow_dx[:, :, :, :].fix(0.0)
+        # accumulation terms
+        # fix accumulation terms to zero and holdup to 1
+        if blk.config.dynamic:
+            # liquid
+            blk.liquid_phase.energy_holdup[:, :, :].fix(1.0)
+            blk.liquid_phase.energy_accumulation[:, :, :].fix(0.0)
+            blk.liquid_phase.material_holdup[:, :, :, :].fix(1.0)
+            blk.liquid_phase.material_accumulation[:, :, :, :].fix(0.0)
+            # vapor
+            blk.vapor_phase.energy_holdup[:, :, :].fix(1.0)
+            blk.vapor_phase.energy_accumulation[:, :, :].fix(0.0)
+            blk.vapor_phase.material_holdup[:, :, :, :].fix(1.0)
+            blk.vapor_phase.material_accumulation[:, :, :, :].fix(0.0)
+
+        # ---------------------------------------------------------------------
+        # get values for state variables for initialization
+        if blk.config.process_type == 'Absorber':
```

Comment:
There is so much automation going on in the background that can easily be omitted in the documentation and has great implications on how a unit model is built and initialized. I do recommend a meeting where these are presented to unit model developers and outcomes from such meetings be updated in the documentation. As seen in this comment, there are alternative ways of calling the initialized method for the properties and the possible implications for each.  The initialized method for the control volume is fixing the end node directly (i.e x[t,0] or x[t,1]) and for now, I don't want that behavior due to numerical issues I have encountered. In the future, if it is resolved, then it can be updated. The user will be supplying values for the duplicate variables (see below) which will be used for the initialization without overriding it. 

```
self.vap_in_flow 
self.vap_in_temperature 
self.vap_in_mole_frac 
self.liq_in_flow 
self.liq_in_temperature
self.liq_in_mole_frac
self.pressure_bottom
```